src/excmds.ts

```diff
@@ -236,6 +236,26 @@ export function im_delete_char() {
     elem.selectionStart = elem.selectionEnd = pos
 }
 
+/**
+ * Within current cursor position of editable HTML area paste our primary selection, thus
+ * expand curent content
+ **/
+//#content
+export function xselpaste(selection: string) {
```

Comment:
We have prefixed other insert mode commands with im_ which I think is really ugly but also a reasonable idea.

Reply:
>im_ which I think is really ugly but also a reasonable idea

I concur. Maybe we could abuse the module exporting mechanism in order to have "insertmode/$nameofthefunction" instead which looks slightly better but might become hard to work around down the line.